app/src/test/java/org/cirdles/topsoil/app/util/AboutDialogViewTest.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2016 CIRDLES.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.cirdles.topsoil.app.util;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.stage.Stage;
+import org.cirdles.topsoil.app.browse.WebBrowser;
+import org.cirdles.topsoil.app.metadata.ApplicationMetadata;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.Mock;
+import static org.mockito.Mockito.when;
+
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import static org.testfx.api.FxAssert.verifyThat;
+import org.testfx.framework.junit.ApplicationTest;
+
+/**
+ *
+ * @author Emily
+ */
+public class AboutDialogViewTest extends ApplicationTest {
+
+    @Rule
+    @SuppressFBWarnings("URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD")
+    public MockitoRule mockitoRule = MockitoJUnit.rule();
+
+    @Mock
+    private ApplicationMetadata metadata;
+
+    @Mock
+    private WebBrowser browser;
+
+    @Override
+    public void start(Stage stage) throws Exception {
+        when(metadata.getVersion()).thenReturn("v1.0.0");
+
+        Parent root = new AboutDialogView(metadata, browser);
+
+        Scene scene = new Scene(root);
+        stage.setScene(scene);
+        stage.show();
+    }
+
+    @Test
+    public void testShowsTopsoilVersion() {
+        verifyThat("#topsoilVersionLabel", (Label label) -> {
+            return label.getText().equals("Product version: v1.0.0");
+        });
+    }
+
+    @Test
+    public void testShowsOperatingSystem() {
+        verifyThat("#operatingSystemLabel", (Label label) -> {
+            boolean isOS = false;
+            if (label.getText().toLowerCase().startsWith("os: mac")) {
+                isOS = true;
+            } else if (label.getText().toLowerCase().startsWith("os: windows")) {
+                isOS = true;
+            } else if (label.getText().toLowerCase().startsWith("os: linux")) {
+                isOS = true;
+            }
+            return isOS;
+        });
```

Comment:
Remember that good code allows for good tests. A good test here would be:

``` java
// Given
// in the @Before
when(systemProperties.getProperty("os.name")).thenReturn("Linux");

// When
// a new about dialog view is shown
// handled by application test

// Then
verifyThat("#operatingSystemLabel", hasText("OS: Linux")));
```

You shouldn't have to branch depending on the operating system. If you feel like you can't write a test without all the `if`s, you should change your code.